### PR TITLE
gtk: Put module of `list_view_apps_launcher` in separate folder

### PR DESCRIPTION
**Pull request by Hofer-Julian**
Seems like I forgot one example

### CODE REVIEW

File: examples/list_view_apps_launcher/application_row/imp.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
+use gtk::{gio, glib};
+
+use gtk::CompositeTemplate;
```
Comment (bilelmoussaoui) [inline examples/list_view_apps_launcher/application_row/imp.rs:5]: I think we can drop this, it's actually re-exported by gtk::subclass::prelude::*;

File: examples/list_view_apps_launcher/application_row/mod.rs [hunk 2]
```diff
@@ -0,0 +1,33 @@
+mod imp;
+
+use gtk::prelude::*;
+use gtk::subclass::prelude::*;
```
Comment (bilelmoussaoui) [inline examples/list_view_apps_launcher/application_row/mod.rs:4]:
```suggestion
```
not needed i guess

File: examples/list_view_apps_launcher/application_row/imp.rs (hunk 1 above)
Comment (Hofer-Julian) [reply examples/list_view_apps_launcher/application_row/imp.rs:5]: It is needed :)

File: examples/list_view_apps_launcher/application_row/mod.rs (hunk 2 above)
Comment (Hofer-Julian) [reply examples/list_view_apps_launcher/application_row/mod.rs:4]: It is needed :)

**Comment by Hofer-Julian** [comment]
Fixed clippy warnings and fixed unrelated flow-box error.
Not sure why the flow-box error was not catched by the CI though.

File: examples/flow_box/main.rs [hunk 3]
```diff
@@ -54,7 +54,7 @@ fn create_color_button(color: &'static str) -> gtk::Button {
     let rgba = gdk::RGBA::from_str(color).unwrap();
     drawing_area.set_draw_func(move |_, cr, _width, _height| {
         GdkCairoContextExt::set_source_rgba(cr, &rgba);
-        cr.paint().expect("Invalid cairo surface state");
```
Comment (bilelmoussaoui) [inline examples/flow_box/main.rs:57]: why ? :p

Comment (bilelmoussaoui) [reply examples/flow_box/main.rs:57]: you need to cargo update & put this back ^^

File: examples/list_view_apps_launcher/application_row/mod.rs (hunk 2 above)
Comment (bilelmoussaoui) [reply examples/list_view_apps_launcher/application_row/mod.rs:4]: ah yes, for ObjectSubclass: -)

File: examples/flow_box/main.rs (hunk 3 above)
Comment (Hofer-Julian) [reply examples/flow_box/main.rs:57]: mmmh, I see.
Will learn at some point to run cargo update every time before I start on a PR

**Pull request merged**